src/track.h

```diff
@@ -29,15 +29,24 @@
 
 extern u8 n_rollovers;
 
+#define SENTINEL_WORDS_EACH 1
```

Comment:
Yes.  I haven't seen any instances of it in 100+ HITL runs, so I'll remove it.  I might make a separate PR to do more general checking of memory getting written where it shouldn't.